### PR TITLE
Syn 288 Created Update Inventory and get order

### PR DESCRIPTION
**Pull request by hapanivivek**

### CODE REVIEW

File: src/main/java/io/github/dftrakesh/zoho/inventory/models/itemapi/updateitem/LineItem.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.Data;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public class LineItem {
+    private String itemId;
```
Comment (manan-dft) [inline src/main/java/io/github/dftrakesh/zoho/inventory/models/itemapi/updateitem/LineItem.java:15]: Arrange the fields properly

Comment (hapanivivek) [reply src/main/java/io/github/dftrakesh/zoho/inventory/models/itemapi/updateitem/LineItem.java:15]: Arranged the code in ascending order

File: src/main/java/io/github/dftrakesh/zoho/inventory/ItemAPI.java [hunk 2]
```diff
@@ -31,4 +33,12 @@ public ItemWrapper getItemById(String itemId, HashMap<String, String> params) {
 
         return getRequestWrapped(request, ItemWrapper.class);
     }
+
+    public UpdateRecordResponse updateItem(HashMap<String, String> params, String updateRecordData) {
```
Comment (manan-dft) [inline src/main/java/io/github/dftrakesh/zoho/inventory/ItemAPI.java]: updateRecordData should be pojo class not string

Comment (hapanivivek) [reply src/main/java/io/github/dftrakesh/zoho/inventory/ItemAPI.java]: Converted to POJO.

**Review by manan-dft — APPROVED** [review]

**Pull request merged**